cfme/tests/containers/test_relationships_tables.py

```diff
@@ -49,19 +50,23 @@ def test_pods_rel(provider, rel):
         lambda ch: 'nginx' not in ch and not ch.startswith('metrics'),
         ui_pods)
 
-    for name in ui_pods_revised:
+    for name in random.shuffle(ui_pods_revised):
         obj = Pod(name, provider)
-
         val = obj.get_detail('Relationships', rel)
-        if val == '0':
-            continue
-        obj.click_element('Relationships', rel)
+        if val != '0':
+            break
```

Comment:
I'm concerned with the impact this has on comparing test results run-over-run, since a different table link could be tested each time. How much time does this save on the test module execution?